README.md

```diff
@@ -20,13 +20,17 @@ The config should like following: (But please don't use the following values for
 
 > Important: The SERVER_SECRET has to have a length of 32 characters!!
 
+<!-- TODO adjust nodes after #106 is done -->
 ````
 PORT=3000
 API_VERSION=v1
 IOTA_PERMA_NODE=https://nodes.iota.org:443
+IOTA_CHRONICLE_NODE=https://chrysalis-nodes.iota.org:443
+IOTA_HORNET_NODE=https://chrysalis-nodes.iota.org:443
+IOTA_STREAMS_NODE=https://api.lb-0.testnet.chrysalis2.com/
```

Comment:
Is it ok that you are using both mainnet and testnet in one project? 
Wouldn't it create a point of failure, if one of the nets (most likely testnet) will undergo some update or maintenance ?

Reply:
yes I've adjusted it to https://chrysalis-nodes.iota.org:443 as well. But this will soon be updated to only have chronicle and hornet node so streams_node and perma_node env var will be removed soon.